### DISCUSSION TITLE
Running sketch locally

### DISCUSSION THREAD

**Discussion by kchusam** [approximatelabs/sketch#30] [Q&A]
Hello,

I tried running sketch locally with the given environment variables but I get this error: 

157 # ``__tracebackhide__`` tells pytest and some other tools to omit this function from tracebacks
    158 `__tracebackhide__ = True`

--> 159 `__pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__`)

ValidationError: 1 validation error for Parameters
stop
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.1/v/missing

Do you have any idea how to solve it?

**Comment by poweress** [comment]
Привет. Я столкнулся с такой же проблемой 
ValidationError: 1 validation error for Parameters
stop
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.4/v/missing

как можно решить эту проблему?

**Comment by bluecoconut** [comment]
This has now been fixed, if you update to `pip install -U sketch` it should work now!
